# Protocal prefix
uint8 VEL = 86 # "V"
uint8 POS = 80 # "P"
uint8 ENC = 69 # "E"

std_msgs/Header header

uint8 predicate
float64[] data